### DISCUSSION TITLE
https://penguins--eggs-net.translate.goog/blog/lastoslinux?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#20] [General]
# https://penguins--eggs-net.translate.goog/blog/lastoslinux?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

LastOSLinux - The methods behind the madness (Custom Distribution Download)

https://penguins--eggs-net.translate.goog/blog/lastoslinux?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

**Comment by LiveFreeDead** [comment]
Thanks, I really am trying to make a change for the better. I am not that experienced at Linux still, but I am improving as I work on this project and sharing the results publicly does cause a lot of negative feedback. But there is some positive feedback too. I am not taking it personally as until I compare my public work to their public work, I don't put much value in the negative reviews - assume they wither don't understand the audience, or just stirring up trouble. Then I also remind myself, I am doing this for free. I'd do it anyway even if it wasn't public. I just hope to help a few people learn along the way.

Having Autism really does make my interests hard to discuss with others - the majority don't know what I am talking about and the ones who do, have their own projects going on. So it's still quite lonely working on things at times. Especially when you sometimes need support on things or feedback on ideas before you commit a heap of time on them. People might not use what your working on anyway - so is it worth writing the manuals, doing the tutorials or sharing anything public? Then again progress is never made by doing nothing, so long as I don't feel it's a waste of time, maybe it wont be :D
